onnx2kerastl/reshape_layers.py

```diff
@@ -540,22 +541,46 @@ def convert_resize(node, params, layers, lambda_func, node_name, keras_name):
         raise ValueError("Invalid axes value")
 
     to_channel_last = keras.layers.Permute((2, 3, 1))(input_tensor)  # (B, W, H, C)
-    shape = tf.cast(tf.shape(to_channel_last), tf.int32)
-    tf_resize_shapes = [shape[i] for i in range(1, 3)]  # (W, H)
+    shape = tf.cast(tf.shape(input_tensor), tf.int32)
```

Comment:
Just noticed that from this line on we exepct the Tensor to be of rank 4 but we don't really test it. 
I think your new fix + old code will fail for Other ranks, WDYT? How about adding
```suggestion
    shape = tf.cast(tf.shape(input_tensor), tf.int32)
    if len(shape) != 4:
       raise Exception("Trying to resize an input tensor of rank !=4 is not yet supported")
    to_channel_last = keras.layers.Permute((2, 3, 1))(input_tensor) # (B, W, H, C)
```